Sources/docc/DocCDocumentation.docc/other-formatting-options.md

```diff
@@ -0,0 +1,108 @@
+# Other Formatting Options
+
+Improve the presentation and structure of your content by incorporating special page elements.
+
+## Overview
+
+DocC offers support for various types of elements such as tables, notes, asides, and special characters.
+
+### Add Tables of Data
+
+To create a table, start a new paragraph and use dashes (`-`) to define 
+columns, and pipes (`|`) as column separators. Construction of a table requires 
+at least three rows: 
+
+* A row that contains the column names.
+* A row with cells that contain only dashes (`-`); three or more for each 
```

Comment:
This text isn't particularly clear. Prefer something like:

A row that consists of pipes and hyphens, used to separate the table header row from the rows of cell contents below.

Reply:
Changed in https://github.com/apple/swift-docc/pull/742/commits/0d0651d7a625986ec00e5b29ecdeae51053d2109